index.html

```diff
@@ -1,15 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="API website" />
     <meta name="keywords" content="HTML, CSS, JavaScript, API" />
     <meta name="author" content="Adriana Stanciu - Elena Marinaki" />
-    <title>API website</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
+    <script defer src="script.js"></script>
+    <title>COVID Tracker</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="center width-sm row app-name">
+        <img src="images/virus.svg" alt="virus" />
+        <h1>COVID TRACKER</h1>
+      </div>
+    </header>
+    <main class="center width-sm stack-lg">
+      <!-- --------------------- COUNTRY SEARCH -->
+      <section>
+        <form class="stack-md">
+          <div>
+            <input type="text" id="country" name="country" />
+          </div>
+          <div class="center row search-label">
+            <label for="country">Search for a country</label>
+            <img src="images/loupe.svg" alt="magnifying glass" />
+          </div>
+        </form>
+      </section>
+      <section>
+        <div class="center row last-visited">
+          <p>Last visited:</p>
```

Comment:
are you thinking of using geolocation? getCurrentPossition?

Reply:
This is very interesting! I haven't thought of it, I was going to search it actually. You mean using the geolocation API, correct? 
That's great actually ✨